test/controllers/session_controller_test.exs

```diff
@@ -0,0 +1,61 @@
+defmodule Xee.SessionControllerTest do
+  use Xee.ConnCase
+  use ExUnit.Case, async: false
+  use Xee.SessionTestHelper, controller: Xee.SessionController
+
+  alias Xee.User
+  alias Xee.Session
+
+  @valid_user_attrs %{name: "valid_user_name", password: "valid_user_password"}
+  @invalid_user_attrs %{name: "invalid_user_name", password: "invalid_user_password"}
+
+  setup do
+    Mix.Tasks.Ecto.Migrate.run(["--all", "Xee.Repo"]);
+    changeset = User.changeset(%User{}, @valid_user_attrs);
+    User.create(changeset, Xee.Repo)
+
+    on_exit fn ->
+      Mix.Tasks.Ecto.Rollback.run(["--all", "Xee.Repo"])
+    end
+  end
+
+  test "GET /login" do
+    conn = get conn(), "/login"
+    assert html_response(conn, 200)
```

Comment:
This line should be like this:
`assert html_response(conn, 200) == "Something"`